.github/workflows/test.yml

```diff
@@ -94,8 +96,8 @@ jobs:
         shell: cmd
         if: matrix.os == 'windows-latest'
 
-  test_ros_binary_install_ubuntu:
-    name: "Check ROS distribution"
+  test_ros_binary_install_linux:
+    name: "Check ROS distribution '${{ matrix.ros_distribution }}'' on '${{ matrix.docker_image }}'"
```

Comment:
```suggestion
    name: "Check ROS distribution '${{ matrix.ros_distribution }}' on '${{ matrix.docker_image }}'"
```

Reply:
Good catch. I am going to commit this manually to have the pre-commit hooks run. Otherwise, this will trigger the linter again.